src/common/common-util/src/main/java/ee/ria/xroad/common/conf/globalconf/VersionedConfigurationDirectory.java

```diff
@@ -200,40 +201,52 @@ private void loadSharedParameters(Path instanceDir, Map<String, SharedParameters
      * Returns private parameters for a given instance identifier.
      * @param instanceId the instance identifier
      * @return private parameters or null, if no private parameters exist for given instance identifier
```

Comment:
Should update javadoc: returns Optional.empty() instead of null. 
The same for findShared() method.

Reply:
Good catch